qutip/correlation.py

```diff
@@ -539,9 +545,9 @@ def spectrum(H, wlist, c_ops, a_op, b_op, solver="es", use_pinv=False):
         print(inspect.stack()[0][3])
 
     if solver == "es":
-        return _spectrum_es(H, wlist, c_ops, a_op, b_op)
+        return _spectrum_es(H, wlist, collapse_ops, A, B)
     elif solver == "pi":
-        return _spectrum_pi(H, wlist, c_ops, a_op, b_op, use_pinv)
+        return _spectrum_pi(H, wlist, collapse_ops, A, B, use_pinv)
```

Comment:
This is one of the lines that is causing the problems in the Travis tests.  The `A` and `B` operators are dense NumPy arrays and not Qobj objects, thus throwing an error.